packages/sources/src/styling/variables/zindex.scss

```diff
@@ -1,5 +1,31 @@
-$zindex-navbar: 1000;
-$zindex-dropdown: 1000;
-$zindex-popover: 1060;
-$zindex-tooltip: 1070;
-$zindex-navbar-fixed: 1030;
+// Selectors
+$zx-viewer-grid-sizing-buttons:     -1;
+$zx-viewer-grid-map-panel:          100;
+
+$zx-selectable:                     100;
```

Comment:
bij `zx-description` en elders kies je er wel voor om de verbijzondering ( `-open` dus) in de naamgeving van de variable op te nemen, maar hier hanteer je alleen de naam van de werkvorm. Liefst 1 lijn kiezen.

Reply:
`zx-description-open` blijft een uitzondering, verder zijn alle verbijzonderingen weg